src/idl_gen_dart.cpp

```diff
@@ -71,7 +71,9 @@ class DartGenerator : public BaseGenerator {
              "// ignore_for_file: unused_import, unused_field, "
              "unused_local_variable\n\n";
 
-      code += "library " + kv->first + ";\n\n";
+      if (kv->first != "") {
```

Comment:
`!kv->first.empty()` ? here and below.

Reply:
ok, fixing

Reply:
> `!kv->first.empty()` ? here and below.

already fixed!

Reply:
> `!kv->first.empty()` ? here and below.

so... is there any other wrong with my code?